### PR TITLE
History updates

### PR DESCRIPTION
**Pull request by kenhunt**
- Moved RespokeClient.getGroupHistories() to use new route. The old route is deprecated.
-  Added RespokeClient.getConversations() and .setConversationsRead() for server-side history.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.1] - 2017-01-10
```
Comment (samuelg) [inline CHANGELOG.md]: Should this be ```1.4.0``` instead of ```1.3.1```?

**Review by samuelg — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (chadxz) [reply CHANGELOG.md]: The formatting could use some love too. The first bullet under added and the first bullet under changed both have a leading `**` without any closing `**` which doesn't bold like it is meant to.

File: RespokeSDK/RespokeClient.m [hunk 2]
```diff
@@ -385,6 +388,112 @@ - (void)getGroupHistoriesForGroupIDs:(NSArray *)groupIDs maxMessages:(NSInteger)
...
+    [signalingChannel sendRESTMessage:@"get" url:urlEndpoint data:nil
+                      responseHandler:^(NSArray* conversations, NSString* errorMessage) {
+                          if (errorMessage) {
+                              errorHandler(errorMessage);
+                              return;
+                          }
+                          
+                          NSMutableDictionary* results = [[NSMutableDictionary alloc] init];
+                          
+                          for (id conv in conversations) {
```
Comment (mbrooks) [inline RespokeSDK/RespokeClient.m]: Any reason for using id here instead of NSDictionary?

File: CHANGELOG.md (hunk 1 above)
Comment (mbrooks) [reply CHANGELOG.md]: I guess technically 1.3.1 could be okay. The API (in theory) should be backwards compatible with 1.3.0, it only adds a new feature.

**Review by mbrooks — APPROVED** [review]
LGTM!

**Pull request merged**